### DISCUSSION TITLE
Postman collection with tests not working

### DISCUSSION THREAD

**Discussion by eben71** [thunderclient/thunder-client-support#1396] [Q&A]
Anyone have examples of how to fix a postman collection containing test scripts?

Currently getting this:
![image](https://github.com/rangav/thunder-client-support/assets/83380241/5f02a18f-2e0f-4837-99f6-9b3ab0afa2b9)

When a script contains something like:
![image](https://github.com/rangav/thunder-client-support/assets/83380241/d2852043-a0e4-454b-878d-d4e371c55ad0)

**Comment by rangav** [comment]
Hi @eben71 

you need to convert postman script to TC script.

Scripting Docs
https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#scripting
